guests_app/src/View/LoginPageView.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {TextInput, Button, SafeAreaView} from 'react-native';
+import {useState} from 'react';
+
+export const LoginPage = (props: any) => {
+	const [phoneNumber, setPhoneNumber] = useState('');
+	const [password, setPassword] = useState('');
```

Comment:
The responsibility of managing user's input should be in the controller.
The view should be responsible only on the presentation of things